website/source/api/nodes.html.md

```diff
@@ -154,103 +154,106 @@ $ curl \
 ```json
 {
   "Attributes": {
-    "driver.rkt": "1",
+    "driver.docker.volumes.enabled": "true",
+    "unique.storage.bytesfree": "8273698816",
+    "driver.mock_driver": "1",
+    "unique.cgroup.mountpoint": "/sys/fs/cgroup",
+    "cpu.arch": "amd64",
+    "consul.revision": "d2adfc0bd",
+    "cpu.numcores": "8",
+    "os.signals": "SIGTTOU,SIGTTIN,SIGSTOP,SIGSYS,SIGXCPU,SIGBUS,SIGKILL,SIGTERM,SIGIOT,SIGILL,SIGIO,SIGQUIT,SIGSEGV,SIGUSR1,SIGXFSZ,SIGCHLD,SIGUSR2,SIGURG,SIGFPE,SIGHUP,SIGINT,SIGPROF,SIGCONT,SIGALRM,SIGPIPE,SIGTRAP,SIGTSTP,SIGWINCH,SIGABRT",
+    "cpu.modelname": "Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz",
+    "nomad.version": "0.10.0-dev",
+    "unique.network.ip-address": "127.0.0.1",
     "driver.docker.bridge_ip": "172.17.0.1",
-    "unique.storage.volume": "/dev/sda1",
-    "driver.exec": "1",
-    "driver.rkt.volumes.enabled": "1",
-    "os.signals": "SIGSTOP,SIGTTIN,SIGWINCH,SIGXCPU,SIGXFSZ,SIGIO,SIGKILL,SIGTTOU,SIGINT,SIGHUP,SIGTRAP,SIGALRM,SIGPIPE,SIGURG,SIGABRT,SIGSEGV,SIGIOT,SIGTERM,SIGBUS,SIGPROF,SIGQUIT,SIGTSTP,SIGUSR2,SIGFPE,SIGCONT,SIGILL,SIGSYS,SIGUSR1,SIGCHLD",
-    "cpu.totalcompute": "2200",
+    "nomad.revision": "30da2b8f6c3aa860113c9d313c695a05eff5bb97+CHANGES",
     "driver.raw_exec": "1",
-    "driver.java.version": "openjdk version \"1.8.0_162",
     "kernel.name": "linux",
-    "unique.cgroup.mountpoint": "/sys/fs/cgroup",
-    "driver.docker.volumes.enabled": "1",
-    "cpu.frequency": "2200",
+    "driver.mock": "true",
```

Comment:
bonus point for sorting these keys in the example by name :)

Reply:
`jq --sort-keys` to the rescue xD